client/widgets/CArtifactsOfHeroBase.cpp

```diff
@@ -257,11 +257,11 @@ void CArtifactsOfHeroBase::setSlotData(ArtPlacePtr artPlace, const ArtifactPosit
 	{
 		artPlace->lockSlot(slotInfo->locked);
 		artPlace->setArtifact(slotInfo->artifact);
-		if(!slotInfo->artifact->canBeDisassembled())
+		if(!slotInfo->artifact->isCombined())
 		{
 			// If the artifact is part of at least one combined artifact, add additional information
 			std::map<const CArtifact*, int> arts;
-			for(const auto combinedArt : slotInfo->artifact->artType->constituentOf)
+			for(const auto combinedArt : slotInfo->artifact->artType->partOf)
```

Comment:
Perhaps add getter method for `partOf`?

Reply:
Will do. VCMI often uses direct access to class fields. If we want to bring VCMI to a uniform style, maybe we should add a rule to the Coding guidelines? Because now the coding style is rather obscure.